### PR TITLE
Add Applitools Eyes tests

### PR DESCRIPTION
**Pull request by bcjordan**
- Add `--eyes` option and `@eyes` tag which filter for only eyes tests
- Run on test server after normal UI test run

image

image

### CODE REVIEW

File: aws/build.rake [hunk 1]
```diff
@@ -213,11 +213,25 @@ $websites_test = build_task('websites-test', [deploy_dir('rebuild')]) do
...
+      end
+
+      HipChat.log 'Running <b>dashboard</b> UI visual tests...'
+      failed_browser_count = RakeUtils.system_ 'bundle', 'exec', './runner.rb', '-c', 'Chrome33Win7', '-d', 'test.learn.code.org', '--eyes'
+      if failed_browser_count == 0
+        message = 'Eyes tests for <b>dashboard</b> succeeded, no changes detected.'
+        HipChat.log message
+        HipChat.developers message, color:'green'
+      else
+        message = 'Eyes tests for <b>dashboard</b> failed. See <a href="https://eyes.applitools.com/app/sessions/">the console</a> for results or to modify baselines.'
```
Comment (laurelfan) [inline aws/build.rake]: do we all need accounts at applitools?

**Comment by laurelfan** [comment]
lgtm!

File: aws/build.rake (hunk 1 above)
Comment (bcjordan) [inline aws/build.rake]: Yup! I'll sort that out & share (& add to getting started doc) this week.

**Comment by bcjordan** [comment]
Merging once secret added

**Pull request merged**